app/services/UserService.py

```diff
@@ -31,3 +31,21 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+async def get_all_users():
```

Comment:
Please specify the return type in the function's prototype.

Reply:
done